website/docs/guide/framework/react.mdx

```diff
@@ -20,7 +20,7 @@ Use with React in a existing Rsbuild project.
 
 To use React with Storybook, you need to install the `storybook-react-rsbuild` package.
 
-<PackageManagerTabs command="storybook-react-rsbuild -D" />
+<PackageManagerTabs command="install storybook-react-rsbuild -D" />
```

Comment:
```suggestion
<PackageManagerTabs command="add storybook-react-rsbuild -D" />
```

We could use `add` as it's easier to type.

Reply:
From what I understand the component only normalizes commands that start with `install` since `yarn` uses `add` but `pnpm` uses `install`

https://github.com/web-infra-dev/rspress/blob/6ceb4b04ba836b36a54f3d4c5cb58b311cafa471/packages/theme-default/src/components/PackageManagerTabs/index.tsx#L25

Reply:
Sorry, I forgot the normalization. LGTM.